crates/infra/cli/src/commands/test/mod.rs

```diff
@@ -46,9 +46,7 @@ fn test_cargo() -> Result<()> {
         .args(["nextest", "run"])
         .flag("--workspace")
         .flag("--all-features")
-        .flag("--lib")
-        .flag("--bins")
-        .flag("--examples")
```

Comment:
I'm not sure we want to remove these other flags. While `--tests` *by default* will include all of these, they can be disabled via `test = false` in the Cargo manifest.